Docs/Configuration.tex

```diff
@@ -2400,6 +2400,17 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
 
   \emph{Note}: This option should be avoided whenever possible. Modern firmware
   usually have compatible AHCI controllers.
+  
+  \item
+  \texttt{ExternalDiskIconsNVME}\\
+  \textbf{Type}: \texttt{plist\ boolean}\\
+  \textbf{Failsafe}: \texttt{false}\\
+  \textbf{Requirement}: 10.4\\
+  \textbf{Description}: Apply icon type patches to IONVMeFamily.kext to force
+  internal disk icons for all NVME disks.
+
+  \emph{Note}: This option should be avoided whenever possible. Modern firmware
```

Comment:
I do not like the description. To get an internal icon displayed you could add an SSDT with a device name and inject a `built-in` device property. This is the correct way to do it. When for some reason (e.g. testing or Thunderbolt) it is difficult, a quirk can be used. This should be explained here.

Reply:
Done